app/configurator/components/breadcrumbs.tsx

```diff
@@ -0,0 +1,45 @@
+// @ts-nocheck
+
+import { Box, BoxProps } from "@theme-ui/components";
+import React from "react";
+
+const Breadcrumbs = ({
+  breadcrumbs,
+  Breadcrumb,
+  onClickBreadcrumb,
+  ...boxProps
+}: {
+  breadcrumbs: any;
+  Breadcrumb: React.ElementType<{ breadcrumb: any; onClick: () => void }>;
+  onClickBreadcrumb?: (i: number) => void;
```

Comment:
nit: Does the `<Breadcrumbs>` component even need to know about how to handle clicks or is it better to leave that to the individual breadcrumbs (e.g. the `<Links>`) and use composition. So this could simply become `{ children: ReactNode } & BoxProps` …

```jsx
// Instead of
<Breadcrumbs
  breadcrumbs={["one", "two"]}
  Breadcrumb={MyComponent}
  onClickBreadcrumb={() => ...}
/>

// Rather this?

<Breadcrumbs>
  <Link href="...">One</Link>
  <Link href="...">Two</Link>
</Breadcrumb>
```

Reply:
Yes you're right I prefer the second way also.